agent/fw_drupal.c

```diff
@@ -327,16 +327,6 @@ NR_PHP_WRAPPER(nr_drupal_http_request_after) {
 }
 NR_PHP_WRAPPER_END
 
-NR_PHP_WRAPPER(nr_drupal_http_request_clean) {
```

Comment:
With this removal, it is failing some drupal integration tests, because the tests were changed for PHP 8.0+ to not create an external segment after an exception. A simple `if` can be added to the "after" instrumentation if this behavior is still desired. Or do we want to go back to what the agent was seemingly previously doing?

(Also, this old "clean" function was leaving a dangling segment on the transaction. Not a memory leak because the transaction handles its own segment memory, but scary nonetheless)

Reply:
Fixed in [9081863](https://github.com/newrelic/newrelic-php-agent/pull/866/commits/9081863a7cd3554ae948f4e3118e91d8fc1b5b29)